### ISSUE TITLE
Support TOML 0.5.0

### ISSUE THREAD

**Issue by Jeehut** [jdfergason/swift-toml#15]
@jdfergason Do you have any plans supporting TOML 0.5.0? According to the docs, it's considered very stable and should be mostly compatible with 1.0 – so it definitely is worth implementing.

**Comment by jdfergason** [comment]
Yes, it should be available in the next release in about a month or so.

**Comment by Jeehut** [comment]
Thank you, I'm looking forward to the next release then! 👍
